draft-sharabayko-mops-srt.md

```diff
@@ -3003,6 +2999,63 @@ IV = (MSB(112, Salt) << 2) XOR (PktSeqNo)
+  packets in order, but based on the timestamps. In the case of a
+  packet arriving too late and skipped by TLPKTDROP mechanism, the
+  order of delivery is still maintained except for potential sequence
+  discontinuity.
+
+  This method has grown historically and is the current common standard
+  for live streaming across different SRT implementations. A change or
+  variation of the settings will break compatibility between two parties.
+
+This combination of settings allows live streaming with a constant latency ({{srt-latency}}). The receiving end will not "fall behind" in time by waiting for missing packets. However, the data integrity might not be ensured, if packets or retransmitted packets do not arrive within the expected time frame. Audio or video interruption can occur, but the overall latency is maintained and does not grow up over time whenever packets are missing.
+
+## File Transmission Use Case {#file-transmission-use-case}
+
+This section describes the use case of file transmission as well as provides configuration examples.
+
+The usage of both message and buffer modes ({{data-transmission-modes}}) is possible in this case. For both modes, Timestamp-Based Packet Delivery (TSBPD) ({{tsbpd}}) and Too-Late Packet Drop (TLPKTDROP) ({{too-late-packet-drop}}) mechanisms must be turned off while congestion control should be set to file transfer congestion control (FileCC) ({{fileCC}).
+
+When TSBPD is disabled, each packet gets the timestamp at the time it is sent by the SRT sender. A packet being sent for the first time will have different timestamp than a corresponding retransmitted packet. In contrast to the live streaming case, the timing of packets' delivery, when sending files, is not critical. The most important thing is data integrity. Therefore TLPKTDROP mechanism must be disabled in this case. No data is allowed to be dropped, because this will result in corrupted files with missing data. The retransmission of missing packets has to happen until the packets are finally acknowledged by the SRT receiver.
+
+The file transfer congestion control (FileCC) will take care of using the available link bandwidth for maximum transfer speed.
+
+### File Transmission in Buffer Mode {#file-transmission-use-case-buffer}
+
+The original UDT protocol {{GHG04b}} used buffer mode ({{transmission-mode-buffer}}) to send files and the same is possible in SRT. The mode was designed to transmit one file per connection. For a single file transmission, a socket is opened, a file is transmitted and then the socket is closed. This procedure is repeated for each single file to follow as the receiver cannot distinguish between two files in a continuous data stream.
+
+The buffer mode is not suitable for the transmission of many small files since for every file a new connection has to be established. To initiate a new connection, at least two round-trip times (RTTs) for the handshake exchange are required ({{handshake-messages}}).
```

Comment:
```suggestion
Buffer mode is not suitable for the transmission of many small files since for every file a new connection has to be established. To initiate a new connection, at least two round-trip times (RTTs) for the handshake exchange are required ({{handshake-messages}}).
```